monitoring/ceph-mixin/tests_alerts/validate_rules.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3 -u
+#!/usr/bin/env python3
```

Comment:
Nice one! I'm afraid we're not consistent with this best practice.

Reply:
Yeah, it didn't work on my system that's why I did it actually :)! (the python of the virtual env generated by tox is not used with the previous shebang)